modules/jta/src/main/java/org/apache/ignite/internal/processors/cache/jta/CacheJtaManager.java

```diff
@@ -37,7 +36,6 @@
  */
 public class CacheJtaManager extends CacheJtaManagerAdapter {
     /** */
```

Comment:
If you remove the constant, you need to remove javadoc to, and new line between class signature and the first constant.